src/style.css

```diff
@@ -1 +1,69 @@
-/* Styles go here */
+body {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0 50px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  box-sizing: border-box;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+ul {
```

Comment:
use classes for styles 
![image](https://user-images.githubusercontent.com/70009205/208705451-60afa693-e33c-4a65-9558-479f50ce287f.png)
fix it everywhere and check yourself with [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)